pybossa/api/project.py

```diff
@@ -60,7 +60,11 @@ def _preprocess_post_data(self, data):
 
 
     def _create_instance_from_request(self, data):
+        if 'password' not in data.keys():
+            raise BadRequest("password required")
```

Comment:
Do we have a test case to cover this exception?

Reply:
Not yet, only got the existing test cases to work, will be adding today.

Reply:
Now added.